tests/test_downloader_handlers.py

```diff
@@ -315,8 +315,8 @@ def test_host_header_seted_in_request_headers(self):
         host = self.host + ':' + str(self.portno)
 
         def _test(response):
-            self.assertEqual(response.body, bytes(host, 'utf-8'))
-            self.assertEqual(request.headers.get('Host'), bytes(host, 'utf-8'))
+            self.assertEqual(response.body, to_bytes(host))
+            self.assertEqual(request.headers.get('Host'), to_bytes(host))
```

Comment:
`to_bytes` is meant for scenarios where the input may be either `bytes` or `str`.

Because here we control the input type, `str`, `host.encode()` would make more sense, being more explicit.

Reply:
Okay